### PR TITLE
Housekeeping | Remove New Relic

### PR DESCRIPTION
**Pull request by gzurowski**
Remove unused New Relic code and dependencies.

### CODE REVIEW

File: src/researchhub/wsgi.py [hunk 1]
```diff
@@ -6,17 +6,14 @@
...
 import os
 
 from django.core.wsgi import get_wsgi_application
 
-CELERY_WORKER = os.environ.get('CELERY_WORKER', False)
-APP_ENV = os.environ.get('APP_ENV') or 'development'
-
-if not CELERY_WORKER and 'development' not in APP_ENV:
-    newrelic.agent.initialize('./newrelic.ini', APP_ENV)
+CELERY_WORKER = os.environ.get("CELERY_WORKER", False)
```
Comment (yattias) [inline src/researchhub/wsgi.py:14]: Seems like `CELERY_WORKER` is used in a couple of places like `wsgi.py`, `settings.py` and `asgi.py`. I'm not sure whether it is needed for celery related purpose

Comment (gzurowski) [reply src/researchhub/wsgi.py:14]: I am not removing this, it just got reformatter (isolated in commit 556e3549a2ff17e85166b0e04c05cf68e183c926).

Comment (yattias) [reply src/researchhub/wsgi.py:14]: Ah oops.

**Review by yattias — APPROVED** [review]
LGTM

**Review by koutst — APPROVED** [review]
LGTM

**Pull request merged**